### PR TITLE
Added heatmap example

### PR DESCRIPTION
**Pull request by TuomasBorman**
Hi,

here is an example of heatmap. 

-Tuomas

### CODE REVIEW

File: 15-microbiome-community.Rmd [hunk 1]
```diff
@@ -15,10 +15,27 @@ data("GlobalPatterns")
...
+Community composition can be visualized with heatmap where other axis represent
+samples and another taxa. Color of each intersection point represent abundance
+of a taxon in a specific sample. 
+
+Here, clr (center log ratio) transformed abundances are plotted at Phylum level. 
+
+```{r heatmap}
+library(ggplot2)
+tse_phylum <- agglomerateByRank(tse, rank = "Phylum")
+tse_phylum <- transformSamples(tse_phylum, method = "clr", pseudocount = 1)
```
Comment (antagomir) [inline 15-microbiome-community.Rmd:29]: I propose adding here transformFeatures(..., method = "Z") as this supports the visualization and is relatively common.

File: 15-microbiome-community.Rmd [hunk 2]
```diff
@@ -15,10 +15,27 @@ data("GlobalPatterns")
...
+
+ggplot(df, aes(x = SampleID, y = FeatureID, fill = clr)) +
+    geom_tile() +
+    scale_fill_gradient(low = "blue", high = "red") + 
+    theme(text = element_text(size=6),
+          axis.text.x = element_text(angle=45, hjust=1)) +
+  labs(x = "Samples", y = "Taxa")
+```
 
 ### Composition barplot
```
Comment (antagomir) [inline 15-microbiome-community.Rmd:40]: This could be moved before the example?

File: 15-microbiome-community.Rmd (hunk 1 above)
Comment (TuomasBorman) [reply 15-microbiome-community.Rmd]: clr --> z

File: 15-microbiome-community.Rmd (hunk 2 above)
Comment (TuomasBorman) [reply 15-microbiome-community.Rmd:20]: Changed the order

File: 15-microbiome-community.Rmd (hunk 1 above)
Comment (antagomir) [reply 15-microbiome-community.Rmd]: clr (on samples) + z (on features)

**Pull request merged**